backend/BackendMIIT/src/main/java/com/example/BackendMIIT/parser/IndividualAchievementsParser.java

```diff
@@ -0,0 +1,105 @@
+package com.example.BackendMIIT.parser;
+
+import com.example.BackendMIIT.config.UrlsConfig;
+import com.example.BackendMIIT.model.domain.IndividualAchievements;
+
+import org.jsoup.Jsoup;
+import org.jsoup.nodes.Document;
+import org.jsoup.nodes.Element;
+import org.jsoup.select.Elements;
+import org.springframework.stereotype.Component;
+
+import java.io.IOException;
+import java.util.HashSet;
+import java.util.Set;
+
+@Component
+public class IndividualAchievementsParser {
+
+    private final UrlsConfig urlsConfig;
+
+    public IndividualAchievementsParser(UrlsConfig urlsConfig) {
+        this.urlsConfig = urlsConfig;
+    }
+
+    public Set<IndividualAchievements> parse() {
+        String iaUrl = urlsConfig.getIa();
+
+        Set<IndividualAchievements> achievements = new HashSet<>();
+
+        try {
+            Document document = Jsoup.connect(iaUrl).get();

```

Comment:
Use SneakyThrows annotation for exceptions

Reply:
Annotated